### ISSUE TITLE
[HTML/CSS] replace <button onclick="window.open(url)"> with link

### ISSUE THREAD

**Issue by Idrinth** [SPIRITORBS/portfolio-HTML-CSS-JS#4]
To make it easier for people with screenreaders as well as webcrawlers linking should be used with a-tags instead. Styling can be applied via css and a class .button for example.

Common button styles feature:
- padding
- font-weight
- box-shadow
- border
- color
- background-color
